ansible_collections/arista/avd/molecule/eos_designs_unit_tests/inventory/group_vars/MH_SERVERS.yml

```
@@ -248,12 +248,16 @@ routers:
               # Flexible encapsulation parameters
               encapsulation_vlan:
                 # client_dot1q: 2101 # default is subinterface number
+              raw_eos_cli: |
```

To avoid the extra blank line
```suggestion
              raw_eos_cli: |-
```